cost/budget_report_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.1
+
+- Removing the 100% limit for the threshold parameter
```

Comment:
This should be in past tense, since it is describing what was done, not what we are doing.

Reply:
Fixed